### ISSUE TITLE
Sensor on a Gigabyte X370 Gaming K7 are not read correctly from the LibireHadwareLib

### ISSUE THREAD

**Issue by KillerGER87** [Rem0o/FanControl.Releases#1654]
My Problem is that LibreHardwareMonitor has a problem with many Gigabyte Mainboards it's seems. Some can at least see the speeds of there Fans some not even that but many can't control them because of a wierd Fancontrol implemantaion from Gigabyte.
But Open Hardware Monitor does manage to Read and Controle my Fans just fine.

So my Question is: would it be possible to the Open Hardware Monitor DLL as a Plugin? 
It is all open Code and Lisence.
[https://github.com/openhardwaremonitor/openhardwaremonitor](Open Hardware Monitor )

**Comment by Rem0o** [comment]
Yes that would be possible, but LHM is a fork of OHM, so compatibility must have been lost along the way.   You could instead try and find what broke by comparing/debugging, and raising a PR in LHM.
